>From 76e63df158315afa2dab5b383fc50ada256a3731 Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <iwamatsu@debian.org>
Date: Sat, 9 Nov 2013 06:32:00 +0900
Subject: [PATCH 4/4] Add alpha, arm64, ppc64 and x32 to list of
 debian_ports_architectures

Signed-off-by: Nobuhiro Iwamatsu <iwamatsu@debian.org>
---
 lib/benl_base.ml | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/lib/benl_base.ml b/lib/benl_base.ml
index d3e0525..d9a31e0 100644
--- a/lib/benl_base.ml
+++ b/lib/benl_base.ml
@@ -61,8 +61,9 @@ let debian_architectures = ref
 let ignored_architectures : string list ref = ref []
 
 let debian_ports_architectures =
-  [ "m68k"; "powerpcspe";
-    "sh4"; "sparc64" ]
+  [ "alpha"; "arm64";
+    "m68k"; "powerpcspe"; "ppc64";
+    "sh4"; "sparc64"; "x32" ]
 
 type status = Unknown | Up_to_date | Outdated
 
-- 
1.8.4.2

